app/views/admin/schedules/_event.html.haml

```diff
@@ -1,14 +1,18 @@
 - cells_length = event.event_type.length / EventType::LENGTH_STEP
 / this height fits the room cells
-- height = (cells_length * 58) - 23
+- compact_grid = EventType::LENGTH_STEP < 15
```

Comment:
My intention here is to keep the original, less dense grids unless the end user really wants to go to more granular schedules.
Regarding the mouseover - this does not solve the issue with touch platforms.
Also for the too-narrow columns with unreadable text - this only applies to the events 15min or less, so it works ok for the longer events